cmd/experimental/kjobctl/pkg/cmd/describe/describe.go

```diff
@@ -198,6 +199,10 @@ func (o *DescribeOptions) Run(ctx context.Context) error {
 		infos = append(infos, configMapsAsInfos...)
 	}
 
+	slices.SortFunc(infos, func(a, b *resource.Info) int {
+		return strings.Compare(a.Name, b.Name)
```

Comment:
is this enough guarantee? Are the names always supposed to match?

Reply:
Yes, we are creating `Job` and `ConfigMap` with the same name.

Reply:
Add a comment to the function indicating why we are sorting and why it works.